tests/opensource-suites/dash/selectors/selectorWithInvalidField.test.ts

```diff
@@ -0,0 +1,30 @@
+import {openTestPage, slct} from '../../../utils';
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+
+import {COMMON_SELECTORS} from '../../../utils/constants';
+import {DashUrls} from '../../../constants/test-entities/dash';
+import DashboardPage from '../../../page-objects/dashboard/DashboardPage';
+import {DashKitOverlayMenuQa, DialogControlQa, SelectQa} from '../../../../src/shared';
+
+datalensTest.describe('Dashboards - Selector behavior with invalid field types', () => {
+    datalensTest(
+        'Should successfully open edit modal when attempting to edit existing selector with incompatible field type',
+        async ({page}) => {
+            await openTestPage(page, DashUrls.DashboardWithInvalidSelector);
+            const dashboardPage = new DashboardPage({page});
+            dashboardPage.forceEnterEditMode();
+
+            await page.click(slct(DashKitOverlayMenuQa.SettingsButton));
+
+            await expect(page.locator(slct('type-error'))).toHaveCount(0);
```

Comment:
Looks like this should be used from common constant for qa attrs 
example - https://github.com/datalens-tech/datalens-ui/blob/main/src/shared/constants/qa/control.ts#L1